mariadb.spec

```diff
@@ -1180,22 +1186,12 @@ fi
 %{_bindir}/myisam_ftdump
 %{_bindir}/myisamlog
 %{_bindir}/myisampack
-%{_bindir}/mysql_convert_table_format
-%{_bindir}/mysql_fix_extensions
 %{_bindir}/mysql_install_db
 %{_bindir}/mysql_secure_installation
-%{_bindir}/mysql_setpermission
 %{_bindir}/mysql_tzinfo_to_sql
-%{_bindir}/mysql_upgrade
-%{_bindir}/mysql_zap
 %{_bindir}/mysqlbug
-%{_bindir}/mysqldumpslow
-%{_bindir}/mysqld_multi
 %{_bindir}/mysqld_safe
-%{_bindir}/mysqlhotcopy
```

Comment:
So between 10.1.20 and 10.1.30, the mysqld_safe_helper binary is (re)introduced (it seems to be present in the 5.5 series also, but is not in 10.1.20), and in 10.1.30 the mysqld_safe script refers to this binary directly.  Therefore either the mariadb-server-utils package must be made a dependency of mariadb-server, or the mysqld_safe_helper file needes to be moved here.